sdk/cognitiveservices/azure-cognitiveservices-inkrecognizer/azure/cognitiveservices/inkrecognizer/_models.py

```diff
@@ -0,0 +1,722 @@
+        """
+        The recognized string of the paragraph. If the words in the paragraph
+        are not recognized, an empty string is returned.
+
+        :rtype: str
+        """
+        return "\n".join([child.recognized_text
+                          for child in self.children])
+
+    @property
+    def lines(self):
+        # type: () -> List[Line]
+        """
+        All the lines in the paragraph.
+
+        :rtype: List[Line]
+        """
+        return [unit for unit in self.children
+                if unit.kind == InkRecognitionUnitKind.LINE]
+
+    @property
+    def list_items(self):
+        # type: () -> List[ListItem]
+        """
+        All the ListItems in the paragraph.
+        """
+
+        return [unit for unit in self.children
+                if unit.kind == InkRecognitionUnitKind.LIST_ITEM]
+
+
+class WritingRegion(InkRecognitionUnit):
+    """
+    A WritingRegion instance represents a certain part of a writing surface that the
+    user has written at least one word on. WritingRegions are the top-level writing
+    objects under an InkRecognitionRoot.
+    """
+    _ink_recognition_unit_kind = InkRecognitionUnitKind.WRITING_REGION
+
+    @property
+    def recognized_text(self):
+        # type: () -> str
+        """
+        The recognized string of the writing region. If the words in the writing
+        region are not recognized, an empty string is returned.
+
+        :rtype: str
+        """
+        return "\n".join([child.recognized_text
+                          for child in self.children])
+
+    @property
+    def paragraphs(self):
+        # type: () -> List[Paragraph]
+        """
+        All paragraphs in the writing region.
+
+        :rtype: List[Paragraph]
+        """
+        return self.children
+
+
+class ListItem(InkRecognitionUnit):
+    """
+    A ListItem instance represents an item of a list. It has a line which has
+    a bullet at the beginning of it.
+    The list that this ListItem belongs to is not given by Ink Recognizer Service.
+    """
+    _ink_recognition_unit_kind = InkRecognitionUnitKind.LIST_ITEM
+
+    @property
+    def recognized_text(self):
+        # type: () -> str
+        """
+        The recognized string of the list item. If the words in the list
+        item are not recognized, an empty string is returned.
+
+        :rtype: str
+        """
+        return "\n".join([child.recognized_text
+                          for child in self.children])
+
+    @property
+    def lines(self):
+        # type: () -> List[Line]
+        """
+        All the lines in the list item (Should be only one).
+
+        :rtype: List[Line]
+        """
+        return self.children
+
+
+class InkRecognitionRoot():
+    """
+    An InkRecognitionRoot instance is the return type from the recognize_ink method in
+    InkRecognizerClient. It is the root of the of recognition unit tree.
+    It also contains the HTTP request status code of the recognition request.
```

Comment:
Why do you need to include the request status code in this model? It will always have the value 200 (or else the method would have raised an exception). 

Unless there is specific need (which I don't see), we should remove the status code from here...

Reply:
Delete it.